lib/gruf/hooks/base.rb

```diff
@@ -45,6 +45,22 @@ def initialize(service, options = {})
       def setup
         # noop
       end
+
+      ##
+      # @return [String] Returns the service name as a translated name separated by periods
+      #
+      def service_key
+        service.class.name.underscore.tr('/', '.')
+      end
+
+      ##
+      # Parse the method signature into a service.method name format
+      #
+      # @return [String] The parsed service method name
+      #
+      def method_key(call_signature)
```

Comment:
Also can we have a better default message? Here is the log I had previously been writing in a custom hook.

```
{
  "params":{},
  "message":"[GRPC::Ok] (health.health_check_service.check)",
  "service":"health.health_check_service",
  "call":"check",
  "grpc_status":"GRPC::Ok",
  "@timestamp":"2017-08-01T15:11:28.432+00:00",
  "@version":"1",
  "severity":"INFO",
  "host":"ffe831f5cb44",
  "thread_id":47383804903340
}
```